README.rst

```diff
@@ -171,7 +171,8 @@ Some extractors require you to provide valid login-credentials in the form of
 a username & password pair.
 This is necessary for ``pixiv``, ``nijie`` and ``seiga``
 and optional (but strongly recommended) for ``exhentai``,  ``luscious``,
-``sankaku``, ``idolcomplex``, ``tsumino`` and ``wallhaven``.
+``sankaku``, ``idolcomplex``, ``tsumino``, ``wallhaven``, ``twitter`` 
```

Comment:
- `wallhaven` doesn't support authentication by username and password any more (see d33f5a7)
- `instagram` got login support with a9c8908